libgenic/pmesh.h

```diff
@@ -16,10 +16,10 @@ typedef struct {
         ptrdiff_t total;
     } ORegion;
     int Nmesh[3];
-} PM;
+} PMField;
```

Comment:
What about PMDef or PMDescriptor / PMDesc? -- PMField smells like an array of double that represents a 3d field.